api.go

```diff
@@ -372,6 +382,7 @@ func (r *Raft) restoreSnapshot() error {
 			r.configurations.latest = configuration
 			r.configurations.latestIndex = snapshot.Index
 		}
+		r.updatePeers()
```

Comment:
This is going to get called on line 332 in `NewRaft()` is there a good reason to call it here? We might change the configuration later when we look through the log entries.